mod-audit-server/src/main/java/org/folio/dao/acquisition/PieceEventsDao.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.dao.acquisition;
+
+import io.vertx.core.Future;
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+import org.folio.rest.jaxrs.model.PieceAuditEvent;
+import org.folio.rest.jaxrs.model.PieceAuditEventCollection;
+
+public interface PieceEventsDao {
+
+  /**
+   * Saves pieceAuditEvent entity to DB
+   *
+   * @param pieceAuditEvent pieceAuditEvent entity to save
+   * @param tenantId        tenant id
+   * @return future with created row
+   */
+  Future<RowSet<Row>> save(PieceAuditEvent pieceAuditEvent, String tenantId);
+
+  /**
+   * Searches for piece audit events by id
+   *
+   * @param pieceId   piece id
+   * @param sortBy    sort by
+   * @param sortOrder sort order
+   * @param limit     limit
+   * @param offset    offset
+   * @param tenantId  tenant id
+   * @return future with PieceAuditEventCollection
+   */
+  Future<PieceAuditEventCollection> getAuditEventsByPieceId(String pieceId, String sortBy, String sortOrder,
+                                                            int limit, int offset, String tenantId);
+
+  Future<PieceAuditEventCollection> getAuditEventsWithUniqueStatusByPieceId(String pieceId, String sortBy, String sortOrder,
```

Comment:
please add java doc to this method as well

Reply:
added java doc